src/executionFiles/bridges/horizon.execute.ts

```diff
@@ -125,133 +119,188 @@ export class HorizonExecutionManager {
       intervalId = setInterval(async () => {
         if (!this.shouldContinue) {
           clearInterval(intervalId)
-          return status
+          return currentExecution
         }
         if (operationId) {
           allowanceAndCrossProcess.operationId = operationId
-          update(status)
+          updateExecution(currentExecution)
           const operation = await bridgeSDK.api.getOperation(operationId)
           console.debug('operation', operation)
 
           // Send > Wait
           if (
             operation.actions[0].status === 'in_progress' &&
-            allowanceAndCrossProcess.status === 'ACTION_REQUIRED'
+            allowanceAndCrossProcess.currentExecution === 'ACTION_REQUIRED'
```

Comment:
Wait, is this refactoring on purpose? The `status` inside the `process` should still be called `status`, right? Only the old `status` object (used in `update(status)`) should be changed. The status inside the process is a string containing the current status and should be still called like that I think

Reply:
Pretty sure this breaks smth 😅